src/python_testing/TC_RR_1_1.py

```diff
@@ -351,6 +355,57 @@ async def test_TC_RR_1_1(self):
         else:
             logging.info("Step 9: Skipped due to no UserLabel cluster instances")
 
+        # Step 10: Count all group cluster instances
+        # and ensure MaxGroupsPerFabric >= 4 * counted_groups_clusters.
+        logging.info("Step 10: Validating groups support minimums")
+        groups_cluster_endpoints: Dict[int, Any] = await dev_ctrl.ReadAttribute(self.dut_node_id, [Clusters.Groups])
```

Comment:
Does it traverse thru all the end points and find out the # group clusters available on the node ?

Reply:
This will produce a dictionary with the endpoint #'s as the key.